src/schema/sale/__tests__/end_sale_mutation.test.js

```diff
@@ -0,0 +1,44 @@
+import { runAuthenticatedQuery } from "test/utils"
+
+describe("End sale mutation", () => {
+  const sale = {
+    id: "foo-foo",
+    _id: "123",
+    currency: "$",
+    is_auction: true,
+    increment_strategy: "default",
+  }
+
+  const query = `
+  mutation {
+    endSale(input: {sale_id: "123"}) {
+      sale {
+        id
+      }
+    }
+  }
+  `
+
+  const rootValue = {
+    saleLoader: sinon.stub().returns(sale),
+    endSaleLoader: sinon.stub().returns(
+      Promise.resolve({
+        sale: {
+          id: "foo-foo",
```

Comment:
Messing with the test realized the returned `id` is coming from `saleLoader` and not `endSaleLoader`. Not sure if the test is wrong or `saleLoader` is not needed [here](https://github.com/artsy/metaphysics/pull/988/files#diff-0d8114e10e8d92ab39dc790190cb4157R24).

Reply:
I believe it is – that's what's used to load data after the mutation is complete.

Reply:
Yea, you can probably drop fetching the sale after the mutation, and get rid of `saleLoader` entirely.

Reply:
What's the convention on what to return once a mutation has completed?

Reply:
Ah nevermind, reading your comments below.